docs_src/security/Ch-SecretStore.md

```diff
@@ -17,1115 +29,259 @@ that you want to tightly control access to, such as API keys, passwords,
-60763abffde3        host                                  host                local
-1d236ab1dbbd        none                                  null                local
-0a7f7266d102        security-secret-store_edgex-network   bridge              local
-```
-
-### Using Consul Web UI
-
-For learning and verification purposes one might use the Consul Web UI
-interface to gather and double check specific Vault informations.
-
-Consul Web UI endpoint port is exposed by the Docker compose file. EdgeX
-Foundry platform uses the Consul default port number 8500. It is
-normally not recommended to expose Consul UI port number in production,
-at least the UI should not be accessible from outside the platform
-environment. However, because all the Vault secrets are encrypted before
-being transmitted and stored in the Consul backend, having access to
-Consul is not sufficient to access any secrets, the vault data
-encryption/decryption key would be absolutely necessary.
-
-Open a Web browser on `http://<EdgeX Consul Server>:8500/ui`.
-
-On the screenshot below, after selecting `SERVICES` and `Vault`, the UI
-will show the various Vault status (heartbeat and init/unseal states),
-coloring the boxes in green, orange or red depending on the level of
-importance (info, warning, error). By clicking each of the right side
-status indicators, more information will be accessible in order to
-better inspect any situation.
-
-![image](EdgeX-UI-Consul-01.png)
-
-As a practical example, we are going to navigate the Consul structure
-for Vault in order to check if the API Gateway (Kong) TLS certificate
-and private key were fetched and stored accordingly during the
-`vault-worker` process.
-
-First select `KEY/VALUE` menu, and then select `vault` root structure:
-
-![image](EdgeX-UI-Consul-02.png)
-
-We are now going to navigate deeper in the `vault` tree structure to
-reach and display the EdgeX Kong TLS assets. Continue by selecting
-`logical/`:
-
-![image](EdgeX-UI-Consul-03.png){.align-center width="348px"
-height="287px"}
-
-Then select `d7809b...` an arbitrary UID generated and created by Consul
-during Vault registration:
-
-![image](EdgeX-UI-Consul-04.png){.align-center width="377px"
-height="216px"}
-
-Select `edgex/`:
-
-![image](EdgeX-UI-Consul-05.png){.align-center width="419px"
-height="228px"}
-
-Select `pki/`:
-
-![image](EdgeX-UI-Consul-06.png){.align-center width="417px"
-height="222px"}
-
-Select `tls/`:
-
-![image](EdgeX-UI-Consul-07.png){.align-center width="418px"
-height="237px"}
+Additionally, Vault supports pluggable "secrets engines".
+EdgeX uses the Consul secrets engine to allow Vault to
+issue Consul access tokens to EdgeX microservices.
 
-Select `edgex-kong/`:
+In EdgeX, Vault's storage backend is the host file system.
+Vault has the capability to run in a high-availability mode
+by supporting replicated storage backends,
+but these high availabilty features are not currently enabled.
```

Comment:
please add why (the reason) HA features are not currently enabled; or just don't mention HA

Reply:
Removed.